src/components/Label/Label.tsx

```diff
@@ -148,41 +154,45 @@ const StyledDiv = styled.div<IStyledLabel>`
   color: ${props => getFontColor(props)};
   font-size: ${props => `${getFontSize(props)}rem`};
   line-height: ${props => `${geLineHeight(props)}rem`};
+  border-radius: ${props => getBorderRadius(props)};
   border: ${props => `1px solid ${getBorderColor(props)}`};
   box-sizing: border-box;
   box-shadow: ${props => getBoxShadow(props)};
+  overflow: hidden;
+  ${props => getInsetStyles(props)}
 `
 
 const LeftIcon = styled(Icon)<IStyledLabel>`
   margin-right: ${props => getIconMargin(props)};
 `
-
 const RightIcon = styled(Icon)<IStyledLabel>`
   margin-left: ${props => getIconMargin(props)};
 `
-
 const Label: ILabel = props => {
   const { className, style, ...rest } = props
-  // For styled components, we separate the props that are to be loaded on the DOM
+  const { text, icons, insetColor } = rest
+  if (insetColor) {
```

Comment:
Hey, can we add a comment to clear out that when insetColor acts as a boolean for whether we can have a custom background or not?

Reply:
Makes sense, added  them.